.values/env/teams/.gitinclude

```diff
@@ -0,0 +1 @@
+*
```

Comment:
why do we need this rule?

Reply:
Placeholder for teams that are empty at start.

Reply:
If it is a place holder then the convention is to use `.gitkeep`